docs/image-builder/README.md

```diff
@@ -0,0 +1,36 @@
+# Use case: Testing packer-plugin-powervs with [image builder](https://github.com/kubernetes-sigs/image-builder)
+
+1. Clone packer-plugin-powervs
+   ```shell
+    $ git clone https://github.com/ppc64le-cloud/packer-plugin-powervs.git
+   ```
+2. Build the binary
+    ```
+   $ cd packer-plugin-powervs
+   $ make build
+   
+   This will build binary named `packer-plugin-powervs` under current directoy.
+   ```
+3. Clone the image-builder
+    ```
+   $ git clone https://github.com/kubernetes-sigs/image-builder.git
+   ```
+4. Install the necessary dependencies for image-builder
+    ```
+   $ cd images/capi
+   $ make deps-powervs
+   ```
+   If the above command returns packages required error manually install them and retry.
+
+
+
+5. Replace the downloaded PowerVS packer plugin with binary built in step 2
+
+   1. When we run ```make deps-powervs```, It will [download and install](https://github.com/Karthik-K-N/image-builder/blob/b8c5d107a92c6d7038823fe7257d9262d49fe36c/images/capi/hack/ensure-powervs.sh#L42-L52) the PowerVS plugin in `"${HOME}/.packer.d/plugins/packer-plugin-powervs"` directory.
+   2. Get the name of the downloaded binary and replace it with binary built in step 2.
+   ```shell
+   $ ls ${HOME}/.packer.d/plugins/packer-plugin-powervs/
+   $ mv /root/packer-plugin-powervs/packer-plugin-powervs ${HOME}/.packer.d/plugins/packer-plugin-powervs/packer-plugin-powervs_v0.2.1_x5.0_linux_amd64
```

Comment:
lets add a note that you are running on x86 server

Reply:
Added a note in the begining of the doc.